backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/humanaethica/assessment/AssessmentController.java

```diff
@@ -34,4 +34,17 @@ public AssessmentDto createAssessment(Principal principal, @PathVariable Integer
         int userId = ((AuthUser) ((Authentication) principal).getPrincipal()).getUser().getId();
         return assessmentService.createAssessment(userId, institutionId, assessmentDto);
     }
+
+    @DeleteMapping("/assessments/{assessmentId}")
+    @PreAuthorize("hasRole('ROLE_VOLUNTEER')")
```

Comment:
This is not enough, it also should be the volunteer that created the assessment. It is necessary to have something like: and hasPermission(#assessmentId, 'ASSESSMENT.WRITER')", and it is necessary to add the new rule to HEPermissionEvaluator.java